src/devices/Ads1115/PgaConfig.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Ads1115
+{
+    /// <summary>
+    ///  Configure the Programmable Gain Amplifier
+    /// </summary>
+    public enum PgaConfig
```

Comment:
Let's change this to also be more descriptive, like ProgrammableGainAmplifier. Also, do you mind adding on the summary above what is this used for? so basically one or two lines saying what will this control?